src/tests/test_webworker_module.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+
+def test_runwebworker_different_package_name(selenium_module_webworker_standalone):
```

Comment:
Rather than duplicating `test_webworker.py` we should use `@pytest.mark.parametrize` to run the same test in both module and classic variants.

Reply:
Good point! Let me see if I can combine them.

Reply:
I ended up creating another fixture `script_type` to create the combination of classic / module vs firefox / chrome. Tried to use `@pytest.mark.parameterize` on the test but it overrides the `request.param` that `selenium_webworker_standalone` fixture already has (firefox/chrome), in which case if we pass the parameters to the test function, we need to pass in all combinations -- classic-firfox / classic-chrome / module-firefox / module-chrome, but that also changes the structure of `request.param`. So it seems creating  another fixture and pass it to the `selenium_webworker_standalone` fixture and test functions is the most straightforward way at the moment. But open to other options if we prefer to modify the `request.params` type.

Reply:
This way is fine with me.